templates/accounts/profile.html

```diff
@@ -5,323 +5,401 @@
+                <div class="panel-body settings-block">
+                    <form method="post">
+                        {% csrf_token %}
+                        <h2>Periodic Reports</h2>
+                        {% if profile.reports_allowed %}
+                            <span class="icon-ok"></span>
+                            Receiving {{period}} reports enabled
+                        {% else %}
+                            <span class="icon-cancel"></span>
+
+                            Receiving reports disabled
+                        {% endif %}
+                        <!-- Button trigger modal -->
+                        <button type="button" class="btn btn-default pull-right" data-toggle="modal"
+                                data-target="#periodModalLong">
+                            Configure...
+                        </button>
+                        <!--<button
+                                name="update_reports_allowed"
+                                type="submit"
+                                class="btn btn-default pull-right">Save
+                        </button>-->
+                    </form>
+
+                    <!-- Modal -->
+                    <div class="modal fade" id="periodModalLong" tabindex="-1" role="dialog"
+                         aria-labelledby="periodModalLongTitle" aria-hidden="true">
+                        <div class="modal-dialog" role="document">
+                            <div class="modal-content">
+                                <div class="modal-header">
+                                    <h3 class="modal-title" id="periodModalLongTitle">Configure your periodic
+                                        reports</h3>
+                                    <!--<button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                      <span aria-hidden="true">&times;</span>
+                                    </button>-->
+                                </div>
+                                <div class="modal-body">
+                                    <form method="post">
+                                        {% csrf_token %}
+
+                                        <label>
+                                            <input
+                                                    name="reports_allowed"
+                                                    type="checkbox"
+                                                    {% if profile.reports_allowed %} checked {% endif %}>
+                                            Send me a summary of my checks
+                                        </label>
+
+                                        <h4>How often do you want to get reports?</h4><br/>
+                                        <label>
+                                            <input
+                                                    name="report_period"
+                                                    type="radio"
```

Comment:
Can you use django forms here for the purpose of learning

Reply:
Working on it